src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -54,6 +57,7 @@ const Body = BaseFormWithPolling.extend({
 
   remove() {
     BaseFormWithPolling.prototype.remove.apply(this, arguments);
+    this.removed = true;
     this.stopProbing();
     this.stopPolling();
   },
```

Comment:
I guess this `remove` method will be called when this courage component is removed from the DOM.
correct me if I am wrong, when the `remove` is called, it only happen in two cases
1. loopback is successful, SIW transit to next view
2. user clicked the cancel (or some other user actions), that SIW load other views

So when 2 happens, we stopped polling, and as consequence, also no need to show the error. I am still not SIW expert, but this apporach makes sense to me

Reply:
Yes, based on the investigation, probe/challenge requests happen in parallel, it can be pretty hard to tell when to stop/abort certain requests, it's ok to let the request fail, in the end what we want is to prevent error message display in another view (after cancel).